packages/node_modules/@webex/webex-core/test/integration/spec/credentials/credentials.js

```diff
@@ -19,6 +19,32 @@ describe('webex-core', () => {
         user = u;
       }));
 
+    describe('#extractOrgIdFromJWT()', () => {
+      let credentials;
+      let webex;
+
+      beforeEach('generate a JWT and Webex Instance', () => {
+        webex = new WebexCore({
+          credentials: user.token
+        });
+
+        credentials = webex.credentials;
+      });
+
+      it('should return the OrgId of the provided JWT', () => {
+        // The access token in a client-auth scenario is a JWT.
+        const token = user.token.access_token;
+
+        assert.equal(credentials.extractOrgIdFromJWT(token), user.orgId);
+      });
+
+      it('should return undefined if the provided JWT is not valid', () =>
```

Comment:
I do like the integration test above to verify the token in a client-auth scenario, but these tests feel like they would be better suited as unit tests.

Reply:
This has been moved.